src/plugins/spark/features/Pid/PidWidget.vue

```diff
@@ -69,7 +69,7 @@ export default class PidWidget extends BlockWidget {
       :actions="[{label:'Enable', handler: enable }]"
       type="info"
       color="info"
-    >This PID is disabled</q-alert>
+    >Driving {{ block.data.outputId }} is disabled</q-alert>
```

Comment:
Change to active speech. Suggestion: "This PID is disabled. outputId is not driven"